### PR TITLE
Redirect to the latest man page when version isn't specified

### PR DESCRIPTION
**Pull request by gustavothecoder**
### What was the end-user problem that led to this PR?

 #618

### What is your fix for the problem, implemented in this PR?

My fix redirects all requests for `/man/*.html` to `/<latest_bundler_version>/man/*.html` using `redirect`.

### Why did you choose this fix out of the possible options?

I chose this fix because it follows the project standards and is very simple.

### CODE REVIEW

**Comment by deivid-rodriguez** [comment]
Thanks @gustavothecoder! CI is failing, can you have a look?

File: config.rb [hunk 1]
```diff
@@ -165,13 +165,24 @@
...
+end
+
 redirect "sponsors.html", to: "https://rubygems.org/pages/sponsors" # Backwards compatibility
 
 page "/conduct.html", layout: :two_column_layout
 page "/compatibility.html", layout: :two_column_layout
 page /\/v(\d+.\d+)\/(?!bundle_|commands|docs|man)(.*)/, layout: :two_column_layout
 page /\/v(.*)\/man\/(.*)/, layout: :two_column_layout
-page /\/man\/(.*)/, layout: :two_column_layout
+page /\/man\/(.*)/, layout: false
```
Comment (gustavothecoder) [inline config.rb:185]: @deivid-rodriguez Now the build is passing 🎉 .

To fix the build, I re-added the man page generic path and removed the layout:
```ruby
page /\/man\/(.*)/, layout: false
```

File: config.rb [hunk 2]
```diff
@@ -165,13 +165,24 @@
...
 end
 
+# Redirect to the latest man page when version isn't specified
+man_files = Dir.glob("./source/#{config[:current_version]}/man/*.html.erb")
+generic_man_urls = man_files.map do |file|
+  file.delete_prefix!("./source/#{config[:current_version]}/")
+  file.delete_suffix!(".erb")
+end
+generic_man_urls.each do |url|
+  latest_man = "#{config[:current_version]}/#{url}"
```
Comment (deivid-rodriguez) [inline config.rb:175]: Does this simplification make sense?

```suggestion
  url = file.delete_prefix("./source/#{config[:current_version]}/").delete_suffix(".erb")
  latest_man = "#{config[:current_version]}/#{url}"
```

Comment (gustavothecoder) [reply config.rb]: Yeah, a lot better.

**Review by deivid-rodriguez — APPROVED** [review]

**Comment by deivid-rodriguez** [comment]
Merging this. Thanks for this great fix @gustavothecoder, it helps contributors to Bundler documentation a lot! 💪

**Comment by tnir** [comment]
Thanks @gustavothecoder for improving the site, but it does not looks like what I meant..., for example https://bundler.io/man/bundle-init.1.html is now in plain text...

## https://bundler.io/man/bundle-init.1.html

bundler io_man_bundle-init 1 html

**Comment by deivid-rodriguez** [comment]
Nice catch @tnir. @gustavothecoder could you have a look at that?

**Comment by tnir** [comment]
@deivid-rodriguez @gustavothecoder Can I revert this as this introduced the bug shown as above?

**Comment by gustavothecoder** [comment]
> Nice catch @tnir. @gustavothecoder could you have a look at that?

Yes, I can.

> @deivid-rodriguez @gustavothecoder Can I revert this as this introduced the bug shown as above?

I think it's a good idea.

**Pull request merged**